Delegate/TaskId.pm

```diff
@@ -1594,6 +1593,7 @@ sub info_user_auth {
             Action.UserLogout
             Model.UserLoginForm
             View.UserAuth->login
+            lockout_mail_task=GENERAL_USER_LOCKOUT_MAIL
```

Comment:
`GENERAL_USER_LOCKED_OUT`. It is not purely a mail task. It renders a page.

I think the names lockout and locked_out  should be merged. Either refer to a LOCKOUT or LOCKED_OUT, everywhere.

Reply:
Yeah, "lockout" is intended to be the verb and "locked out" is the adjective, but I understand how it's confusing as they are too similar sounding.